public/js/controllers/MainCtrl.js

```diff
@@ -57,6 +57,22 @@ angular.module('MainModule', []).controller('MainController',['$scope', 'StockSe
     );
   }
 
+  $scope.isStockPriceEmpty = function(){
+    if($scope.currentCompany == "Your Portfolio"){
+      if($scope.superStockPriceHistory.length>0)
+        return false;
+      else
+        return true;
```

Comment:
Not a big fan of if (something) { return true/false } else { return false/true }....i think it's better to just return the value of the something itself

return $scope.superStockPriceHistory.length <= 0